### PR TITLE
Update to Angular 14 + Eslint

### PR DESCRIPTION
**Pull request by teknekale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
-Decorator-powered REST client for Angular 12 and its HttpClient, plus RxJs 6+.
+Decorator-powered REST client for **Angular 9+** and its HttpClient, plus **RxJs 6+**.
 
-For previous versions of Angular use **Grappa** `v0.8.3`.
+| Last version | Angular Versions       | Node             |
+|--------------|------------------------|------------------|
+| `1.0.0`      | 9+ up to 14 (included) | 14.15.0 or later |
 
-## Installation
+## 🛠 Installation
```
Comment (dvidgry) [inline README.md:9]: 🛠️ 😎

File: README.md [hunk 2]
```diff
@@ -1,22 +1,15 @@
 # Grappa
 
-Decorator-powered REST client for Angular 12 and its HttpClient, plus RxJs 6+.
+Decorator-powered REST client for **Angular 9+** and its HttpClient, plus **RxJs 6+**.
 
-For previous versions of Angular use **Grappa** `v0.8.3`.
+| Last version | Angular Versions       | Node             |
+|--------------|------------------------|------------------|
+| `1.0.0`      | 9+ up to 14 (included) | 14.15.0 or later |
```
Comment (Auxx) [inline README.md:7]: That's NOT last version.

File: README.md (hunk 1 above)
Comment (teknekale) [reply README.md:9]: :)

**Review by Auxx — CHANGES_REQUESTED** [review]

**Review by Auxx — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (teknekale) [reply README.md]: rolled back to `0.8.5`

**Pull request merged**